packages/qqwing.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Qqwing < Package
+  description 'QQwing is a Sudoku puzzle generator and solver.'
+  homepage 'https://qqwing.com/'
+  @_ver = '1.3.4'
+  version @_ver
+  source_url "https://github.com/stephenostermiller/qqwing/archive/v#{@_ver}.tar.gz"
+  source_sha256 'dc5d61b4470862b190d437f8143d4090639c164d34461c4caa5c081c5c91e6cc'
+
+  def self.patch
+    system 'sed -i s:@build/src-copyright-check.sh::g Makefile'
+    system 'echo > build/src_neaten.pl && chmod +x build/src_neaten.pl'
+  end
+    
+  def self.build
+    system 'make', "prefix=#{CREW_PREFIX}", 'tgz'
+  end
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    FileUtils.mkdir_p CREW_DEST_LIB_PREFIX
+    FileUtils.ln_s "#{CREW_PREFIX}/lib/libqqwing.so", "#{CREW_DEST_LIB_PREFIX}/libqqwing.so"
+    FileUtils.ln_s "#{CREW_PREFIX}/lib/libqqwing.so", "#{CREW_DEST_LIB_PREFIX}/libqqwing.so.2"
+    FileUtils.ln_s "#{CREW_PREFIX}/lib/libqqwing.so", "#{CREW_DEST_LIB_PREFIX}/libqqwing.so.2.1.0"
+    FileUtils.ln_s "#{CREW_PREFIX}/lib/libqqwing.la", "#{CREW_DEST_LIB_PREFIX}/libqqwing.la"
```

Comment:
Would it make sense to adjust the Makefile to put the libraries in the correct location.  In any event, this will crash on all non-x86_64 architectures since `#{CREW_PREFIX}/lib` is the same as `#{CREW_DEST_LIB_PREFIX}` since it will try to symlink to the same location.

Reply:
You also don't need to include `libqqwing.la`.  That file is only needed during the build.